src/Core/src/AwsClientFactory.php

```diff
@@ -178,6 +179,19 @@ public function ecr(): EcrClient
         return $this->serviceCache[__METHOD__];
     }
 
+    public function elastiCache(): ElastiCacheClient
```

Comment:
Missing CHANGELOG entry.

Should we add a test in the CI for that?

Reply:
Oh. Thank you. 

Hm... maybe. But I will not add that test now.